docs/guides.md

```diff
@@ -38,6 +38,29 @@ If all your VMs are "agile", that is, they're not tied to local storage or local
 
 3. (After a reboot or host startup) **Move VMs back** to the host if appropriate. There is no need to re-enable the host: it is done automatically when it starts.
 
+## Logging system
+
+### logrotate
+
+`logrotate` is the tool to administrate the rotation, compression, removal, ... of log files.
+The configuration is located in `/etc/logrotate.conf` and is used by the daily cron task `/etc/cron.daily/logrotate`.
+
+By default a file is rotated if:
+  - its size is greater than 100 MiB (since 8.2.1 version)
+  - it's a new day (daily rotation)
+
+Also a file is compressed after two rotations, the first time it is just renamed.
+
+### rsyslog
+
+Because a file must be rotated if a log exceeds 100 MiB, the `rsyslog` daemon is used to trigger automatically the `/etc/cron.daily/logrotate` without waiting. (Conf location: `/etc/rsyslog.d/xenserver.conf`)
```

Comment:
```suggestion
Because a file must be rotated if a log exceeds 100 MiB, the `rsyslog` daemon is used to trigger automatically the `/etc/cron.daily/logrotate`command  without waiting for the logrotate cron job to run. (Conf location: `/etc/rsyslog.d/xenserver.conf`)
```